File: app/src/main/java/ru/eva/oriokslive/models/miet/news/Enclosure.java
package ru.eva.oriokslive.models.miet.news;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;

@Root(name = "enclosure", strict = false)
public class Enclosure {

    @Attribute(name = "url")
    private String url;

    public String getUrl() {
        return url;
    }
}

File: app/src/main/java/ru/eva/oriokslive/network/entity/news/Enclosure.kt
package ru.eva.oriokslive.network.entity.news

import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Root

@Root(name = "enclosure", strict = false)
data class Enclosure(
    @field:Attribute(name = "url")
    @param:Attribute(name = "url")
    val url: String,
)